app/static/styles/main.css

```diff
@@ -41,6 +57,23 @@ body {
   height: 60px;
 }
 
+.nav__links {
+  list-style: none;
+}
+
+.nav__links li {
+  display: inline-block;
+  padding: 0px 20px;
+}
+
+
+.nav__links li a:hover {
+
+color: var(--yellow)
+}
+
+
+
```

Comment:
On another note, I think we should stick to a uniform naming convention for CSS. All existing styles use `dash-names` :))